cocos/tween/actions/action.ts

```diff
@@ -194,6 +196,14 @@ export abstract class Action {
         this.tag = tag;
     }
 
+    setId (id: number): void {
```

Comment:
It is better to check id first.

Reply:
But the game code may also invoke this function too.

Reply:
Updated, id 's default value is undefined now. And all numbers are treated as valid values.